addons/dialogue_nodes/nodes/forkNode.gd

```diff
@@ -0,0 +1,299 @@
+@tool
+extends GraphNode
+##
+## Fork Node
+##
+## Groups a list of output options, each with a condition. The first condition (top to
+## bottom) to be valid is used to exit, with a default option with no conditions always last.
+
+signal modified
+signal disconnection_from_request(from_node : String, from_port : int)
+signal connection_shift_request(from_node : String, old_port : int, new_port : int)
+
+@export var max_options := -1
+@export var color_option : Color = Color.GREEN_YELLOW
+@export var color_default : Color = Color.INDIAN_RED
+
+var undo_redo : EditorUndoRedoManager
+var last_size := size
+
+var options: Array = []
+var empty_option : BoxContainer
+var option_height: int = 0
+
+@onready var title_label: LineEdit = %Title
+@onready var prev_title_text := title_label.text
+
+@onready var first_option_index: int = %ForkOption1.get_index()
+@onready var default_option: BoxContainer = %DefaultForkOption
+
+@onready var orig_height: int = size.y  # Includes 2 options (starter + default)
+
+@onready var OptionScene := preload("res://addons/dialogue_nodes/nodes/sub_nodes/forkOption.tscn")
```

Comment:
Please change the scene name to `ForkOption.tscn` here.
Also nit: the project uses single quotes ' everywhere so please change the string accordingly.

Reply:
Name changed. Changed all `""` inside Fork scripts into `''`.